docker-compose.yml

```diff
@@ -1,23 +1,17 @@
 services:
   app:
-    image: ims-nest-api-starter
     container_name: ims-nest-api-starter
     build:
       context: .
       dockerfile: Dockerfile
+    env_file: .env.docker
     ports:
       - "8000:8000"
     volumes:
-      - .:/var/www/ims-nest-api-starter
-      - node_modules:/var/www/ims-nest-api-starter/node_modules

```

Comment:
this was needed for the issue created with `bcrypt`, did you manage to solve this issue?